src/pspm_cfg/pspm_cfg_help_format.m

```diff
@@ -39,7 +39,7 @@
     evalc(sprintf('helptext = settings.help.%s.Arguments.%s;', funcname, argname));
     % remove entries in square brackets
     [startindx, endindx] = regexp(helptext, '\[\s*([^\[\]]*)\s*\]'); % thanks ChatGPT for finding the regexp
-    for k = 1:numel(startindex)
+    for k = numel(startindx):-1:1
```

Comment:
the previous version will not work if `numel(stardindx) > 1` because elements of the string are deleted in the loop and this would change the index of later elements. Also the variable `startindx` was misspelt.